pybossa/data_access.py

```diff
@@ -198,6 +198,5 @@ def set_default_amp_store(project):
     annotation_config = project['info'].get('annotation_config', {})
     input_data = project['info'].get('data_classification', {}).get('input_data') or ''
     output_data = project['info'].get('data_classification', {}).get('output_data') or ''
-    # Set amp_store default to False if 'L1' is in input or output data classifications.
```

Comment:
Why do we remove the default amp_store? I thought we only want to remove default settings when syncing, but when user create project through API or other method, we need the default amp_store (opt-out for L1, and opt-in for L2-L4). Am I missing anything?

Reply:
I guess the main issue before this change was that when a L2-L4 project with no "amp_pvf" configured, synchronizing project to Prod with amp_store True results an error, causing the sync failed. See Paxton's comment in the PR for detail.